.github/workflows/ci.yml

```diff
@@ -57,11 +57,16 @@ jobs:
   min-version:
     name: min-version
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # Current MSRV and expected next MSRV
+        RUST: ["1.48.0", "1.56.0"]
```

Comment:
Is 1.56 actually an expected next MSRV? that was IIRC just the one that one of those dependency minor versions required. I don't think it was any more intentional than that.

Reply:
It's pyca/cryptography's next expected MSRV (based on our usage numbers). So we'd be happy with that reality :-)

Reply:
Fair enough!